backend/LeetWars.Core/LeetWars.Core.Common/DTO/Challenge/BriefChallengeInfoDto.cs

```diff
@@ -0,0 +1,14 @@
+﻿using LeetWars.Core.Common.DTO.User;
+using LeetWars.Core.DAL.Entities;
+
+namespace LeetWars.Core.Common.DTO.Challenge
+{
+    public class BriefChallengeInfoDto : Entity<long>
```

Comment:
Entity<T> is used to represent the Id of the entity, so you should omit ChallengeId property